### PR TITLE
Sundry changes as requested

### PR DESCRIPTION
**Pull request by gburnhill**

### CODE REVIEW

File: HelpMyStreetFE/HelpMyStreetFE/src/sass/views/_content-page.scss [hunk 1]
```diff
@@ -36,6 +36,10 @@
         li:before {content: counters(item, ".") " "; counter-increment: item}
     }
 
+  &__body {
+      font-size: 16px;
```
Comment (mike-f50) [inline HelpMyStreetFE/HelpMyStreetFE/src/sass/views/_content-page.scss:40]: We have a rule `* { font-size: 18px; }` (which I dislike, and) which will apply to all child elements of `.content-page__body`.  I therefore wouldn't expect this new rule to have any effect.

If you want to override this you'd need `content-page__body * { font-size: 16px; }` - but bear in mind the Resources page has this class too, and you probably don't want to change the size of the headings there.  Maybe define a more specific class `.body-text-16px`, with css something like the following:

`
/* Applies to any p, ol or ul which is a child of a .body-text-16px, and additionally and children of those elements */
.body-text-16px {
  p, p *, ol, ol *, ul, ul * {
    font-size: 16px;
  }
}
`

(Apparently line breaks don't persist in this thing, sorry!)

Comment (mike-f50) [reply HelpMyStreetFE/HelpMyStreetFE/src/sass/views/_content-page.scss:40]: image

**Comment by gburnhill** [comment]
It seems to work though...

Sent from my iPhone

Geoff Burnhill | Doctor | Musician

On 16 Oct 2020, at 19:04, mike-f50 <notifications@github.com> wrote:

﻿

@mike-f50 commented on this pull request.

________________________________

In HelpMyStreetFE/HelpMyStreetFE/src/sass/views/_content-page.scss<https://github.com/HelpMyStreet/front-end/pull/492#discussion_r506639568>:

> @@ -36,6 +36,10 @@
         li:before {content: counters(item, ".") " "; counter-increment: item}
     }

+  &__body {
+      font-size: 16px;


We have a rule * { font-size: 18px; } (which I dislike, and) which will apply to all child elements of .content-page__body. I therefore wouldn't expect this new rule to have any effect.

If you want to override this you'd need content-page__body * { font-size: 16px; } - but bear in mind the Resources page has this class too, and you probably don't want to change the size of the headings there. Maybe define a more specific class .body-text-16px, with css something like the following:

/* Applies to any p, ol or ul which is a child of a .body-text-16px, and additionally and children of those elements */ .body-text-16px { p, p *, ol, ol *, ul, ul * { font-size: 16px; } }

(Apparently line breaks don't persist in this thing, sorry!)

________________________________

In HelpMyStreetFE/HelpMyStreetFE/src/sass/views/_content-page.scss<https://github.com/HelpMyStreet/front-end/pull/492#discussion_r506640455>:

> @@ -36,6 +36,10 @@
         li:before {content: counters(item, ".") " "; counter-increment: item}
     }

+  &__body {
+      font-size: 16px;


[image]<https://user-images.githubusercontent.com/58040070/96293185-46378000-0fe2-11eb-97f0-2bbfb0060347.png>

**Comment by mike-f50** [comment]
Really?  
image

**Comment by gburnhill** [comment]
Ha, I am unsurprisingly a fool.
Updated as per your recommend.

On 19 Oct 2020, at 09:28, mike-f50 <notifications@github.com<mailto:notifications@github.com>> wrote:



Really?
[image]<https://user-images.githubusercontent.com/58040070/96420473-7606a380-11ed-11eb-83de-7dbef7b20562.png>

**Review by mike-f50 — APPROVED** [review]

**Pull request merged**